certdb/sql/database_accessor.go

```diff
@@ -202,6 +206,21 @@ func (d *Accessor) GetRevokedAndUnexpiredCertificatesByLabel(label string) (crs
 	return crs, nil
 }
 
+// GetRevokedAndUnexpiredCertificatesSelectColumnsByLabel gets serial_number and revoed_at from all revoked and unexpired certificate from db (for CRLs) with specified ca_label.
+func (d *Accessor) GetRevokedAndUnexpiredCertificatesByLabelSelectColumns(label string) (crs []certdb.CertificateRecord, err error) {
```

Comment:
do you think it makes sense to make another struct that's a subset of `certdb.CertificateRecord` that way someone doesn't try to use one of the other cols in the future (which will always be empty)